config/initializers/wings.rb

```diff
@@ -3,23 +3,14 @@
 
 Rails.application.config.after_initialize do
   # Add all concerns that are migrating from ActiveFedora here
-  CONCERNS = [ConferenceItem, Dataset, ExamPaper, GenericWork, Image, JournalArticle, PublishedWork, Thesis].freeze
+  CONCERNS = [ConferenceItem, Dataset, Etd, ExamPaper, GenericWork, Image, JournalArticle, PublishedWork, Thesis].freeze
```

Comment:
Should Oer need to be in here too?

Reply:
At this point, we should leave OER and ETD out because those two work types aren't functioning right for adventist and that's a separate ticket to fix them